notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<ol><li>You can briefly explain what the repos are. E.g. DATASET_IN is where the text data is, DATASET_OUT is where we'll store the embeddings.</li><li>Normally, it's good to define the global variables all at the beginning, but I think it may make the notebook a bit easier to follow if the variables related to Inference Endpoint creation are declared in that section. </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>